src/components/CalendarLayout/_components/CalendarList/CalendarList.tsx

```diff
@@ -111,20 +70,27 @@ export function CalendarList(props: ICalendarListProps) {
 					locale={{
 						emptyText: 'Aucun rendez-vous',
 					}}
-					dataSource={sortedAndFilteredAppointments}
+					dataSource={futureAppointments}
 					renderItem={(item) => (
-						<List.Item key={item.school_name}>
+						<List.Item key={item.school_name?.toString() || ''}>
 							<List.Item.Meta
 								className="favorites-list__item"
-								title={item.school_name}
+								title={item.school_name?.toString() || ''}
 								description={
 									<Flex justify="space-between">
 										<i>
-											{item.school_city} -{' '}
+											{item.school_address?.toString() || ''}{' '}
+											{item.school_postal_code?.toString() || ''}{' '}
+											{item.school_city?.toString() || ''} -
+											{getNameFromEmail(item.users?.email || '').name}
 											<div>
-												{item.planned_date ? dayjs(item.planned_date).format('HH:mm') : 'N/A'}
+												{item.planned_date
+													? dayjs(item.planned_date.toString()).format('HH:mm')
+													: 'N/A'}
 											</div>
-											{item.planned_date ? dayjs(item.planned_date).format('DD/MM/YYYY') : 'N/A'}
+											{item.planned_date
+												? dayjs(item.planned_date.toString()).format('DD/MM/YYYY')
+												: 'N/A'}
```

Comment:
S'il n'y a pas de dates, autant ne rien afficher.
```suggestion
											{item.planned_date
												?? dayjs(item.planned_date.toString()).format('DD/MM/YYYY')
												}
```